core/src/api/oas-loader.ts

```diff
@@ -42,14 +46,48 @@ const selectSecurities = ({
   });
 };
 
+const parseJSON = async (filename: string) => {
+  const rawText: string = await fs.readFile(filename, "utf8");
```

Comment:
Currently only supports local file, need remote file support to be consistent with SwaggerParser